src/sp/it/pl/util/action/Action.java

```diff
@@ -637,7 +637,7 @@ public static void installActions(Object... os) {
 
 	public static void loadCommandActions() {
 		// discover all command actions defined in file
-		File file = new File(APP.DIR_USERDATA, "command-actions.cfg");
```

Comment:
This is not correct, because the file is not well-formed xml due to allowing variable declarations. It is possible to use xml comments for that purpose, but currently it works differently, using # character, which is simpler to parse and more importantly write/read by a human.

CoreSerializerXml either needs to be adjusted for xml comments (which would push additional complexity to user) or the file can not be an xml. I prefer the latter of course.

Reply:
but I even found an Xml parser call in your code? And it improves highlighting. That Xml format isn'T really convenient for the user as well, if it's about that, then you should probably prefer json

Reply:
Yes, but xml it is for now, unless someone recodes it. I dont believe you have the actual file because the user directory is in gitignore, but this would break my scrips actually.

Reply:
I think the scripts in the file will no longer work.

Reply:
But your player shouldn't care about the extension, it processes the file the same no matter how its named?

Reply:
.xml means the file is xml but it clearly isn't. This is wrong.